package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "freedom",
   "description": "Embracing a distributed web",
-  "version": "0.6.25",
+  "version": "0.6.26",
```

Comment:
What about in the other repos where they depend on this?  e.g. https://github.com/freedomjs/freedom-for-firefox/pull/70/files?  Should I just undo all the changes to package.json and expect that the ~ and ^ will pick up the latest version?